openhcl/bootloader_fdt_parser/src/lib.rs

```diff
@@ -488,6 +498,7 @@ impl ParsedBootDtInfo {
                     config_ranges = info.config_ranges;
                     partition_memory_map = info.partition_memory_map;
                     accepted_ranges = info.accepted_memory;
+                    vtl0_alias_map = info.vtl0_alias_map;
```

Comment:
Maybe it's time to destructure the return struct of parse_openhcl?

Reply:
Yeah probably a good idea.

(I don't really see why we have all these layers to begin with. This all seems way too complicated to me. But I don't have time to propose a solution right now.)